### ISSUE TITLE
Smart Deploy – Deploy without Setup

### ISSUE THREAD

**Issue by timbrandin** [arunoda/meteor-up-legacy#634]
Just an idea, do we really need the setup step? Can't we just have deploy be smart and install new versions of node, mongo if the current meteor versions require that and all that?

I mean just do `meteor up deploy` and you're all set.

**Comment by KrishnaPG** [comment]
+1 Thats a good method indeed.

**Comment by 4shaw** [comment]
Yeah good idea... One issue I have is when I "mupx setup" on the same server all the other meteor apps loose their connection to the database... That will have to be fixed else every time you deploy you will have to restart all the other meteor apps on that server... Its a rather painful issue... Anyone know why that happens?
